packages/quantic/force-app/main/default/lwc/quanticSmartSnippet/quanticSmartSnippet.js

```diff
@@ -332,19 +333,11 @@ export default class QuanticSmartSnippet extends LightningElement {
   }
 
   /**
-   * Returns the smart snippet source title.
-   * @returns {string}
-   */
-  get sourceTitle() {
-    return this?.state?.source?.title;
-  }
-
-  /**
-   * Returns the smart snippet source uri.
-   * @returns {string}
+   * Returns the smart snippet source.
+   * @returns {Result}
```

Comment:
Needs to be a clearer type.

Otherwise, call it `result` if it's really the result.

Reply:
It's is really a result indeed: https://github.com/coveo/ui-kit/blob/5dfc1437d1080cbeb74df7dbbcad99a6b6a0f17e/packages/headless/src/controllers/core/smart-snippet/headless-core-smart-snippet.ts#L182-L185

and we are calling it `source` in Headless as it is a smart snippet source so that's why I kept the same name.